src/ApiCore/AuthWrapper.php

```diff
@@ -55,6 +63,63 @@ public function __construct(FetchAuthTokenInterface $fetchAuthTokenInterface, ca
         $this->authHttpHandler = $authHttpHandler;
     }
 
+    /**
+     * @param array $args {
+     *     An array of required and optional arguments.
+     *
+     *     @type CredentialsLoader $credentialsLoader
```

Comment:
Should `credentialsLoader` and `authHttpHandler` have "Optional" in the description like some of the other fields?

Reply:
I have done a bit of a restructuring of this method and moved the logic to check for `credentialsLoader` into the `GapicClientTrait`, PTAL